conference.js

```diff
@@ -650,31 +650,13 @@ export default {
         // overshadowed by the overlay.
         tryCreateLocalTracks.then(tracks => {
             const isGranted = (type) => tracks.some(track => track.type === type);
-            const isAudioGranted = isGranted('audio');
-            const isVideoGranted = isGranted('video');
 
             APP.store.dispatch(toggleSlowGUMOverlay(false));
             APP.store.dispatch(notifyMediaPermissionsGranted({
-                audio: isAudioGranted,
-                video: isVideoGranted
+                audio: isGranted('audio'),
+                video: isGranted('video')
             }));
-
-            if (tracks.length >= 2) {
```

Comment:
If this condition does not evaluate true then there are not local audio+video track in some cases, maybe because this code runs too early. Hard to figure out why this is without a local setup.